interview-questions.md

```diff
@@ -8,32 +8,36 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the virtual DOM in React?
 
-Your answer:
+Your answer: A virtual DOM is either the displayed local window page (3000) or something to do with the UI where the person inputs info and interacts with the app/site.
 
-Researched answer:
+Researched answer: A Javascript object that essentially act as a mirror, or branch of the real DOM. In the same way a branch is used during the GIT/GIT HUB cycle to work on instead of altering the main until approved/merged. The virtual DOM does the same. 
 
 2. Ruby has an implicit return. What does that mean?
 
-Your answer:
+Your answer: A shortcut(s) ?
```

Comment:
Excellent work in this section, especially on your researched answers.  Try to avoid one or two word sentences as an answer in an interview whenever possible.  Try to expand, even if that means using examples.